tests/foreman/cli/test_jobtemplate.py

```diff
@@ -28,183 +28,178 @@
 from robottelo.cli.factory import make_org
 from robottelo.cli.job_template import JobTemplate
 from robottelo.datafactory import invalid_values_list
-from robottelo.test import CLITestCase
 
 TEMPLATE_FILE = 'template_file.txt'
 TEMPLATE_FILE_EMPTY = 'template_file_empty.txt'
+ssh.command(f'''echo '<%= input("command") %>' > {TEMPLATE_FILE}''')
+ssh.command(f'touch {TEMPLATE_FILE_EMPTY}')
 
 
-class JobTemplateTestCase(CLITestCase):
-    """Implements job template tests in CLI."""
+@pytest.mark.tier1
+def test_positive_create_job_template(module_org):
+    """Create a simple Job Template
 
-    @classmethod
-    def setUpClass(cls):
-        """Create an organization to be reused in tests."""
-        super().setUpClass()
-        cls.organization = make_org()
-        ssh.command(f'''echo '<%= input("command") %>' > {TEMPLATE_FILE}''')
-        ssh.command(f'touch {TEMPLATE_FILE_EMPTY}')
+    :id: a5a67b10-61b0-4362-b671-9d9f095c452c
 
-    @pytest.mark.tier1
-    def test_positive_create_job_template(self):
-        """Create a simple Job Template
+    :expectedresults: The job template was successfully created
 
-        :id: a5a67b10-61b0-4362-b671-9d9f095c452c
+    :CaseImportance: Critical
+    """
+    template_name = gen_string('alpha', 7)
+    make_job_template(
+        {
+            'organizations': module_org.name,
+            'name': template_name,
+            'file': TEMPLATE_FILE,
+        }
+    )
+    assert JobTemplate.info({'name': template_name}) is not None
 
-        :expectedresults: The job template was successfully created
 
-        :CaseImportance: Critical
-        """
-        template_name = gen_string('alpha', 7)
-        make_job_template(
-            {
-                'organizations': self.organization['name'],
-                'name': template_name,
-                'file': TEMPLATE_FILE,
-            }
-        )
-        self.assertIsNotNone(JobTemplate.info({'name': template_name}))
-
-    @pytest.mark.tier1
-    def test_negative_create_job_template_with_invalid_name(self):
-        """Create Job Template with invalid name
-
-        :id: eb51afd4-e7b3-42c3-81c3-6e18ef3d7efe
-
-        :expectedresults: Job Template with invalid name cannot be created and
-            error is raised
+@pytest.mark.tier1
+def test_negative_create_job_template_with_invalid_name(module_org):
```

Comment:
@pondrejk should we parameterize this test?
`@pytest.mark.parametrize('name', **parametrized(invalid_values_list()))`

Reply:
I would agree, so that one name failure doesn't block testing of the rest.